### DISCUSSION TITLE
Donde Encuentro Las Practicas

### DISCUSSION THREAD

**Discussion by allanbre95** [Launch-X-Latam/MisionFrontEnd#141] [General]
Hola!

He participado de las 4 sesiones que hemos tenido hasta ahora pero no vi mencion de la parte de las practicas ni donde las encuentro, alguien me puede ayudar a salir de esa duda por favor?

Gracias!!

**Comment by CarlosAldanaCh** [comment]
Aquí las encuentras compañero. https://github.com/Launch-X-Latam/MisionFrontEnd/blob/5641f91e814d46f40cc9121fe922fb46f932d3a0/01%20-%20INTRO/practicas/README.md

**Reply by allanbre95** [reply]
Muchas gracias!

**Comment by ghost** [comment]
Aquí están

https://github.com/Launch-X-Latam/MisionFrontEnd/blob/main/01%20-%20INTRO/practicas/README.md

**Reply by JorgeDevmm** [reply]
Hasta cuando es la fecha de entrega compañero
